pokecli.py

```diff
@@ -116,6 +116,7 @@ def get_commit_hash():
 
                 while True:
                     bot.tick()
+                    inventory.refresh_inventory()
```

Comment:
An other solution could be to add something like ```inventory.to_json()```. This way we wouldn’t need to make a call to the server.
But this seems pretty annoying to do. Definitely not on top of my priority list, I won’t do it myself.